tests/iac/test_models.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from ggshield.iac.models import (
+    IaCMatch,
+    IaCMatchSchema,
+    IaCPolicyBreak,
+    IaCPolicyBreakSchema,
+    IaCScanResult,
+    IaCScanResultSchema,
+)
+
+
+class TestModel:
+    @pytest.mark.parametrize(
+        "schema_klass, expected_klass, instance_data",
+        [
+            (
+                IaCMatchSchema,
+                IaCMatch,
+                {"filename": "hello", "match": "hello", "type": "hello"},
+            ),
+            (
+                IaCPolicyBreakSchema,
+                IaCPolicyBreak,
+                {
+                    "type": "hello",
+                    "policy": "hello",
+                    "validity": "hey",
+                    "matches": [
+                        {"filename": "hello", "match": "hello", "type": "hello"}
+                    ],
+                    "policy_details": {},
+                },
+            ),
+            (
+                IaCScanResultSchema,
+                IaCScanResult,
+                {
+                    "policy_break_count": 1,
+                    "policies": ["pol"],
+                    "policy_breaks": [
+                        {
+                            "type": "break",
+                            "policy": "mypol",
+                            "matches": [
+                                {
+                                    "filename": "hello",
+                                    "match": "hello",
+                                    "type": "hello",
+                                }
+                            ],
+                            "policy_details": {},
+                        }
+                    ],
+                },
+            ),
+        ],
+    )
+    def test_schema_loads(self, schema_klass, expected_klass, instance_data):
+        """
+        GIVEN the right kwargs  and an extra field in dict format
+        WHEN loading using the schema
+        THEN the extra field should be excluded
```

Comment:
Not very clear, do you mean here "the extra field is not taken into account" ? Otherwise I don't see a test that the `extra` field is excluded from the models ?

Reply:
This test is the same as the one in pygitguardian. Basically, it doesn't add the extra field to the instance when deserializing the object